handlers/handlers.go

```
@@ -51,10 +70,39 @@ func HandleMessageEvent(v *events.Message, db *sql.DB, client *whatsmeow.Client)
+	defer cancel()
+
+	resp, err := ai.GenerateReply(ctx, msgText, evt.Info.Sender.String())
+	if err != nil {
+		fmt.Printf("AI reply error: %v\n", err)
+		return
+	}
+	if !resp.ShouldReply || strings.TrimSpace(resp.Reply) == "" {
+		return // not laundry-related — skip, don't reply
+	}
+
+	msg := &waProto.Message{Conversation: proto.String(resp.Reply)}
```

golangci-lint (2.12.2)

[error] SA1019: waProto.Message is deprecated: use new packages directly 

(staticcheck)